client-templates/container-registry-agent/accept.json.sample

```diff
@@ -3,7 +3,17 @@
   {
     "//": "image scan callback (v1)",
     "method": "POST",
-      "path": "/api/v1/import/app"
+    "path": "/api/v1/import/app"
+  },
+  {
+    "//": "image scan callback (v1)",
+    "method": "POST",
+    "path": "/api/v1/import/scan-result"
+  },
+  {
+    "//": "image scan callback (v1)",
```

Comment:
Would more detail be useful on these? All are `image scan callback (v1)` but calling different endpoints.

Reply:
@carlos-snyk added some more detail, but nowI need you to approve again 🙂